### ISSUE TITLE
 I only got the accuarcy of 94.6%, not 94.8% on SIW-13

### ISSUE THREAD

**Issue by lulu1214** [lluisgomez/script_identification#4]
Dear lluisgomez,  I only got the accuarcy of 94.6%, not 94.8% on SIW-13. I followd all steps according to your reference.  Have you use some initial parameters for CNN? ?

**Comment by lluisgomez** [comment]
The results shown on the paper are obtained following the same steps, but due to stochastic GD and random sampling of the training set you may get a slightly different accuracy.
